### ISSUE TITLE
Error loading mnist ONNX model

### ISSUE THREAD

**Issue by lauracanalini** [deephealthproject/eddl#150]
Hi,
we are trying to use onnx to save the eddl models, so we tried to run `2_onnx_save_net_to_file.cpp` and load the file saved with `3_onnx_import_net_from_file.cpp`.
During loading we get this error:
```
Ir_version = 6
Operator domain  = 
Operator version = 11
Producer_name: EDDL
Producer_version: 0.1
Domain: 
Model_version: 0
Net imported from ONNX succesfully
Generating Random Table
Selecting GPU device 0
EDDLL is running on GPU device 0, GeForce RTX 2080 Ti
CuBlas initialized on GPU device 0, GeForce RTX 2080 Ti
CuRand initialized on GPU device 0, GeForce RTX 2080 Ti
terminate called after throwing an instance of 'std::runtime_error'
  what():  Incorrect padding type (PoolDescriptor::PoolDescriptor)
Aborted (core dumped)
```
System information:
- Ubuntu 19.04
- GCC 8.3.0
- eddl master branch (commit d89e085aed9bdb45d5b0b23fc9e46a87943795cb)

Does this also happen to you?

**Comment by chavicoski** [comment]
Can you share the code or the topology of the network?

**Comment by lauracanalini** [comment]
We're trying to use your examples, so the network is the one in `eddl/examples/onnx/2_onnx_save_net_to_file.cpp`.

```
// Define network
    layer in = Input({784});
    layer l = in;  // Aux var

	 l = l=Reshape(l,{1,28,28});
	 l=MaxPool(ReLu(BatchNormalization(Conv(l,32,{3,3},{1,1}))),{2,2});
	 l=MaxPool(ReLu(BatchNormalization(Conv(l,64,{3,3},{1,1}))),{2,2});
	 l=MaxPool(ReLu(BatchNormalization(Conv(l,128,{3,3},{1,1}))),{2,2});
					 
	 l=Flatten(l);
									 
	 l=Activation(Dense(l,256),"relu");
											 
	 l=Activation(Dense(l,128),"relu");
													 
	 layer out=Activation(Dense(l,num_classes),"softmax");
															 
	 // Net define input and output layers list
	 model net=Model({in},{out});
```

**Comment by prittt** [comment]
Hey @chavicoski, is there any news from your side?

**Comment by chavicoski** [comment]
Not yet, we are working on it.

**Comment by prittt** [comment]
Ok, thank you. Let us know!

**Comment by RParedesPalacios** [comment]
Hi, 
So people here tell me that it is working... Are you using the las Master version?
Regards

**Comment by RParedesPalacios** [comment]
Hi again,
I have clone Master. Compile everything. and run:

./onnx_export 
./onnx_import 

and this is the result:

./onnx_export 
Generating Random Table
CS with low memory setup
---------------------------------------------
input1                        |  (784)     =>      (784)     
reshape1                      |  (784)     =>      (784)     
dense1                        |  (784)     =>      (1024)    
relu1                         |  (1024)    =>      (1024)    
batchnorm1                    |  (1024)    =>      (1024)    
dense2                        |  (1024)    =>      (1024)    
relu2                         |  (1024)    =>      (1024)    
batchnorm2                    |  (1024)    =>      (1024)    
dense3                        |  (1024)    =>      (1024)    
relu3                         |  (1024)    =>      (1024)    
batchnorm3                    |  (1024)    =>      (1024)    
dense4                        |  (1024)    =>      (10)      
softmax4                      |  (10)      =>      (10)      
---------------------------------------------

1 epochs of 600 batches of size 100
Epoch 1
Batch 600 softmax4(soft_cross_entropy=0.058,categorical_accuracy=0.918) 0.104 secs/batch
62.649 secs/epoch
Evaluate with batch size 100
Batch 100 softmax4(soft_cross_entropy=0.021,categorical_accuracy=0.964) 
Saved net to onnx file


./onnx_import 
Ir_version = 6
Producer_name: EDDL
Producer_version: 0.1
Domain: 
Model_version: 0
Net imported from ONNX succesfuly
Generating Random Table
Output size list = 1
CS with low memory setup
---------------------------------------------
input1                        |  (784)     =>      (784)     
reshape1                      |  (784)     =>      (784)     
dense1                        |  (784)     =>      (1024)    
relu1                         |  (1024)    =>      (1024)    
batchnorm1                    |  (1024)    =>      (1024)    
dense2                        |  (1024)    =>      (1024)    
relu2                         |  (1024)    =>      (1024)    
batchnorm2                    |  (1024)    =>      (1024)    
dense3                        |  (1024)    =>      (1024)    
relu3                         |  (1024)    =>      (1024)    
batchnorm3                    |  (1024)    =>      (1024)    
dense4                        |  (1024)    =>      (10)      
softmax4                      |  (10)      =>      (10)      
---------------------------------------------

Evaluate with batch size 100
Batch 100 softmax4(soft_cross_entropy=0.021,categorical_accuracy=0.964)

**Comment by chavicoski** [comment]
The error is already fixed and uploaded to master. You can try to run it again
